Services/src/main/resources/webutils/templates/webutils-templates.html

```diff
@@ -1655,6 +1756,46 @@ <h4 class="modal-title">${attr('title')}</h4>
 			}, {"element": fileLink, "$scope" : context.$scope, "actionHelper" : context.actionHelper, "modelName": modelName}));
 		</script>
 	</attr-directive>
+	
+	<attr-directive name="ykOnDrag">
+		<script>
+			var funName = context.attributeValue;
+			
+			var element = context.element;
+			
+			$(element).off("dragstart").on("dragstart", $.proxy(function(event){
+				var methodName = eval("this.$scope." + this.funName);

```

Comment:
Here the result of eval is not methodName. I think it was added by mistake. Remove it.